apps/server/package.json

```diff
@@ -0,0 +1,65 @@
+{
+	"name": "server",
```

Comment:
working on the assumption this gets appended by pnpm to the name in the root manifest?